### ISSUE TITLE
[Newbie Question About fasterrcnn_resnet50_fpn]There are no outputs when evaluating on validation set

### ISSUE THREAD

**Issue by weirdseed** [pytorch/vision#2155]
I trained my dataset for several times on training set using pretrained weights, but there are no values in validation outputs. `[{'boxes': tensor([], device='cuda:0', size=(0, 4), grad_fn=<StackBackward>), 'labels': tensor([], device='cuda:0', dtype=torch.int64), 'scores': tensor([], device='cuda:0', grad_fn=<IndexBackward>)}, {'boxes': tensor([], device='cuda:0', size=(0, 4), grad_fn=<StackBackward>), 'labels': tensor([], device='cuda:0', dtype=torch.int64), 'scores': tensor([], device='cuda:0', grad_fn=<IndexBackward>)}]`

**Comment by mthrok** [comment]
Hi

Could you provide a minimal complete code example so that we can reproduce this?

**Comment by weirdseed** [comment]
> Hi
> 
> Could you provide a minimal complete code example so that we can reproduce this?

Thanks. I think I found the problem, and it's solved!
